br/pkg/task/restore.go

```diff
@@ -733,18 +734,26 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 	return nil
 }
 
-func runRestore(c context.Context, mgr *conn.Mgr, g glue.Glue, cmdName string, cfg *RestoreConfig, checkInfo *PiTRTaskInfo) error {
+func runSnapshotRestore(c context.Context, mgr *conn.Mgr, g glue.Glue, cmdName string, cfg *RestoreConfig, checkInfo *PiTRTaskInfo) error {
 	cfg.Adjust()
 	defer summary.Summary(cmdName)
 	ctx, cancel := context.WithCancel(c)
 	defer cancel()
 
+	log.Info("starting snapshot restore")
 	if span := opentracing.SpanFromContext(ctx); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("task.RunRestore", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
 		ctx = opentracing.ContextWithSpan(ctx, span1)
 	}
 
+	keepaliveCfg := GetKeepalive(&cfg.Config)
+	// Restore needs domain to do DDL.
+	mgr, err := NewMgr(ctx, g, cfg.PD, cfg.TLS, keepaliveCfg, cfg.CheckRequirements, true, conn.NormalVersionChecker)
```

Comment:
It seems we already have a `mgr`, why reopen it here? If we need a `mgr` with domain, I think we'd better let the caller pass such one.

Reply:
ah let me recheck the diff. This is a mistake when doing rebasing. I removed the domain var and inlined in the mgr creation, but it had conflicts with master and accidentally added again